Stresser/ProcessArtifact.cpp

```diff
@@ -3,9 +3,10 @@
 ProcessArtifact::ProcessArtifact(const std::string& name, const std::string& data) : IArtifact(name, data)
 {
     std::wstring tempPath = LocalPcUtils::getTempPath();
-    std::wstring wideFileName = std::wstring(ATL::CA2W(name.c_str()));
+    std::wstring wideFileName = std::wstring(ATL::CA2W(data.c_str()));
```

Comment:
Suggestions: Add util in `StringUtils`, thus you can convert from `stringToWString` and from `wstringToString`